### ISSUE TITLE
[Bug]: Freetube blackscreen after being launched

### ISSUE THREAD

**Issue by KobraRocks** [FreeTubeApp/FreeTube#2197]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.

### Describe the bug

Since v.0.16.0, after launching Freetube  the app does not load the home screen.

### Expected Behavior

the app launches and show the home screen

### Issue Labels

causes crash

### FreeTube Version

v0.16.0

### Operating System Version

Garuda Linux - White-tailed-eagle (Arch Based)

### Installation Method (If applicable)

other

### Last Known Working FreeTube Version (If Any)

v0.15.1

### Additional Information

Error log:
`[4062763:0413/221421.696450:ERROR:vaapi_wrapper.cc(1136)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[4062763:0413/221421.696570:ERROR:vaapi_wrapper.cc(1083)] FillProfileInfo_Locked failed for va_profile VAProfileH264Main and entrypoint VAEntrypointVLD
[4062763:0413/221421.696632:ERROR:vaapi_wrapper.cc(1136)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[4062763:0413/221421.696677:ERROR:vaapi_wrapper.cc(1083)] FillProfileInfo_Locked failed for va_profile VAProfileH264High and entrypoint VAEntrypointVLD
/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../include/c++/11.2.0/bits/unique_ptr.h:659: typename std::add_lvalue_reference<element_type>::type std::unique_ptr<blink::CSSSelector []>::operator[](std::size_t) const [_Tp = blink::CSSSelector [], _Dp = std::default_delete<blink::CSSSelector []>]: Assertion 'get() != pointer()' failed.
[4062683:0413/221422.505858:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.DBus.StartServiceByName: object_path= /org/freedesktop/DBus: org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked thereply, the reply timeout expired, or the network connection was broken.`

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Did u get FT from AUR?

**Comment by KobraRocks** [comment]
I've got it from chaotic-aur, referenced under freetube 0.16.0-1
It depends on  gtk3, nss and electron

I have tested the AUR package freetube-bin 0.16.0-1 and it work right out of the box

error message is now: 
`[160680:0414/073729.040932:ERROR:vaapi_wrapper.cc(1123)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[160680:0414/073729.041050:ERROR:vaapi_wrapper.cc(1070)] FillProfileInfo_Locked failed for va_profile VAProfileH264Main and entrypoint VAEntrypointVLD
[160680:0414/073729.041103:ERROR:vaapi_wrapper.cc(1123)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[160680:0414/073729.041142:ERROR:vaapi_wrapper.cc(1070)] FillProfileInfo_Locked failed for va_profile VAProfileH264High and entrypoint VAEntrypointVLD
closed`

**Comment by absidue** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/2163
see this comment for more information: https://github.com/FreeTubeApp/FreeTube/issues/2163#issuecomment-1092683219

**Comment by KobraRocks** [comment]
Ok thx for pointing me to the right place.
